zcash_client_sqlite/src/wallet/scanning.rs

```diff
@@ -1501,7 +1507,8 @@ pub(crate) mod tests {
         //   in the last note in that block.
         // * The next block crosses the shard boundary, with two notes in the prior
         //   shard and two blocks in the subsequent shard
-        // * An additional 100 blocks are scanned, to ensure that
+        // * An additional 100 blocks are scanned, to ensure that the checkpoint
+        //   is pruned.
```

Comment:
Nit: this change, and the two below it, can safely be moved to the third commit that added the test (as they are not changes forced by the `shardtree` bump).